mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/Connection.cs

```diff
@@ -2119,7 +2119,9 @@ internal void SetSocketTimeouts (int send_timeout, int receive_timeout, int keep
 		}
 
 		internal long Thread_GetElapsedTime (long id) {
-			return SendReceive (CommandSet.THREAD, (int)CmdThread.GET_ELAPSED_TIME, new PacketWriter ().WriteId (id)).ReadLong ();
+			if (Version.AtLeast (2, 50))
```

Comment:
This should be checked in ThreadMirror.ElapsedTime instead using vm.CheckProtocolVersion (), so calling the public method will fail.